homeassistant/components/application_credentials/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Application Credentials",
   "config_flow": false,
   "documentation": "https://www.home-assistant.io/integrations/application_credentials",
-  "dependencies": ["auth", "websocket_api"],
+  "dependencies": ["auth", "cloud", "websocket_api"],
```

Comment:
Application credentials shouldn't depend on cloud. Something else is the reason for the issue, I think.

Reply:
Ah, that makes sense